src/sgame/sg_cmds.cpp

```diff
@@ -575,6 +575,12 @@ void Cmd_Give_f( gentity_t *ent )
 		G_AddMomentumGeneric( (team_t) ent->client->pers.team, amount );
 	}
 
+	if ( Q_strnicmp( name, "bp", strlen("bp") ) == 0 )
+	{
+		float bp = trap_Argc() < 3 ? 300.0f : atof( name + strlen("bp") );
```

Comment:
As I understand, more narrowly scoped variables are usually preferable. `amount` is only there at the top because of C89.